streaming/src/main/scala/org/apache/spark/streaming/dstream/TransformedDStream.scala

```diff
@@ -51,4 +51,15 @@ class TransformedDStream[U: ClassTag] (
     }
     Some(transformedRDD)
   }
+
+  /**
+   * Wrap a body of code such that the call site and operation scope
+   * information are passed to the RDDs created in this body properly.
+   * This has been overriden to make sure that `hideInnerRDDOps` is always `true`, that is,
+   * the inner scopes and callsites of RDDs are always exposed.
+   */
+  override protected[streaming] def createRDDWithLocalProperties[U](
+      time: Time, hideInnerRDDOps: Boolean)(body: => U): U = {
```

Comment:
hideInnerRDDOps not used? Also can you have either hide or display? It's confusing to have both.